devenv/loadtest/render_test.js

```diff
@@ -1,33 +1,58 @@
-import { check, group } from 'k6';
+import { check, fail, group } from 'k6';
 import { createClient } from './modules/client.js';
+import {
+  createTestOrgIfNotExists,
+  upsertTestdataDatasource,
+  upsertTestdataDashboard,
```

Comment:
```suggestion
  upsertDashboard,
```
If you agree with my suggestion above